### DISCUSSION TITLE
Is the measure name the same as the column name

### DISCUSSION THREAD

**Discussion by mlonsk** [TabularEditor/TabularEditor3#1172] [Resources]
@pawarbi has created a nice little script for identifying which measures use the same name as the columns of a table. 

This can, as he writes, be helpful when using a measure table where there are no restrictions on which names a measure can have. 

Check it out in [his blog](https://fabric.guru/c-script-to-get-a-list-of-measures-matching-column-names)

**Comment by pawarbi** [comment]
Thank you for sharing
